gym-connect4/gym_connect4/envs/lib/connect4.py

```diff
@@ -1,21 +1,34 @@
 import board
 import move
+from ai import Ai
+import human
 class Connect4():
 
     def __init__(self):
         self.game_board = board.Board()
         self.moves = self.game_board.moves
+        self.player1 = Ai(self.moves)
+        self.player2 = Ai(self.moves)
+
+    def test(self, input_values):
+        def mock_input(s):
```

Comment:
s is the array we provide in the test, and print prints out the first value popped?